### PR TITLE
[build] add sonic-mgmt docker to PR test

### PR DESCRIPTION
**Pull request by yxieca**
#### Why I did it
Found regression on sonic-mgmt docker build.

#### How I did it
Add PR test to catch regression.

#### How to verify it
PR test.

### CODE REVIEW

**Comment by yxieca** [comment]
docker-ptf is already built in PR test currently. And the change didn't get sonic-mgmt-docker to build. Investigation is needed.

**Comment by yxieca** [comment]
VS build failure is expected. After #7009 merged, retest will pass.

File: .azure-pipelines/build-template.yml [hunk 1]
```diff
@@ -92,6 +92,7 @@ jobs:
         fi
 
         make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/docker-sonic-vs.gz target/sonic-vs.img.gz target/docker-ptf.gz
+        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/docker-sonic-mgmt.gz
```
Comment (lguohan) [inline .azure-pipelines/build-template.yml:95]: this increase the vs job time, which is the critical path for running test. suggest to have a new job then.

**Pull request closed without merging**